pkgs/by-name/pw/pwalarmd/package.nix

```diff
@@ -0,0 +1,35 @@
+{ lib
+, fetchFromGitHub
+, rustPlatform
+, pkg-config
+, alsa-lib }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "pwalarmd";
+  version = "0.1.0";
+
+  nativeBuildInputs = [ pkg-config ];
+  buildInputs = [ alsa-lib ];
+
+  src = fetchFromGitHub {
+    owner = "amyipdev";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-xoC1PtDQjkvoWb9x8A43ITo6xyYOv9hxH2pxiZBBvKI=";
+  };
+
+  cargoHash = "sha256-cRAFnmgvzWLFAjB7H1rU4FdxMwm0J6d76kdFPoXpPMw=";
+
+  meta = with lib; {
```

Comment:
`with lib;` is antipattern. Use lib.licenses lib.maintainers

Reply:
I just got my patches merged for libitl that used `with lib;`...

Reply:
There are many PR that are trying to remove `with lib;` See https://github.com/NixOS/nixpkgs/issues/208242